packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/ExtractEquationsResponseHandler.java

```diff
@@ -0,0 +1,26 @@
+package software.uncharted.terarium.hmiserver.service.tasks;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class ExtractEquationsResponseHandler extends TaskResponseHandler {
+
+	public static final String NAME = "nougat_task:extraction";
```

Comment:
Are we only using nougat to extract equations, or is it going to be used for something else?
I found the task name `nougat_task:extraction` too vague compare to the class name `ExtractEquations...`